app/Console/RemoveFacilitiesFromPlace.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Silex\Console;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Offer\Commands\UpdateFacilities;
+use CultuurNet\UDB3\Search\ResultsGenerator;
+use CultuurNet\UDB3\Search\ResultsGeneratorInterface;
+use CultuurNet\UDB3\Search\SearchServiceInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+final class RemoveFacilitiesFromPlace extends AbstractCommand
+{
+    private const FACILITY_QUERY = 'terms.id:3.';
```

Comment:
Are you sure this will work? I get 0 results on production for this `q`, even with `disableDefaultFilters=true`: 

https://search.uitdatabank.be/places/?disableDefaultFilters=true&q=terms.id:3.

Reply:
It does work if you add a wildcard at the end it seems: [https://search.uitdatabank.be/places/?disableDefaultFilters=true&q=terms.id:3.*](https://search.uitdatabank.be/places/?disableDefaultFilters=true&q=terms.id:3.*)

Reply:
Indeed! I forgot the wildcard.